src/main/java/org/asciidoc/intellij/annotator/ExternalAnnotator.java

```diff
@@ -115,7 +115,7 @@ public void apply(@NotNull PsiFile file, AsciiDocAnnotationResultType annotation
       }
       if (logRecord.getMessage().startsWith("possible invalid reference:")) {
         /* TODO: these messages are not helpful in IntelliJ as they have no line number
-           and for splitted documents they provide too many false positives */
+             and provide too many false positives for split documents  */
```

Comment:
Additional spacing to make TODO multi-line aware, see [uing-todo](https://www.jetbrains.com/help/idea/using-todo.html)

Reply:
reverted to only one space indent, see commit 4c72fad